docs/yaml_docs/pipeline_preprocess_yml.md

```diff
@@ -0,0 +1,437 @@
+                          # then use this section of the modality dictionary
+                          # in this case any obs['is_doublet'] that are False will be retained in the dataset.
+```
+
+<span class="parameter">filtering</span><br>
+  - <span class="parameter">run</span> `Boolean`, Default: True<br>
+    If set to False, no filtering is applied to the `MuData` object.
+
+  - <span class="parameter">keep_barcodes</span> `String` (Path)<br>
+    Path to a file containing specific cell barcodes you want to keep; leave blank if not applicable.
+
+### RNA-specific filtering (rna)
+<span class="parameter">obs</span><br>
+    Parameters for obs, i.e. cell level filtering:
+
+  - <span class="parameter">min</span><br>
+    Filtering cells based on a minimum value in a column. Leave parameters blank if you do not want to filter by them.
+
+    - <span class="parameter">n_genes_by_counts</span> `Integer`<br>
+      Minimum number of genes by counts per cell.
+      For instance, setting the parameter to 500, will filter out cells with a value less than 500 in the n_genes_by_counts column.
+
+  - <span class="parameter">max</span><br>
+    Filtering cells based on a maximum value in a column. Leave parameters blank if you do not want to filter by them.
+    
+    - <span class="parameter">total_counts</span> `Integer`<br>
+      Cells with a total count greater than this value will be filtered out.
+    
+    - <span class="parameter">n_genes_by_counts</span> `Integer`<br>
+      Maximum number of genes by counts per cell.
+
+    - <span class="parameter">pct_counts_mt</span> `Integer` (in Percent)<br>
+      Percent of counts that are mitochondrial genes. Cells with a value greater than this will be filtered out.
+      Should be a value between 0 and 100 (%).
+    
+    - <span class="parameter">pct_counts_rp</span> `Integer` (in Percent)<br>
+      Percent of counts that are ribosomal genes. Cells with a value greater than this will be filtered out.
+      Should be a value between 0 and 100 (%).
+
+    - <span class="parameter">doublet_scores</span> `Integer`<br>
+      If you want to apply a custom scrublet threshold per input sample you can specify it here.
+      Provide either as one score for all samples (e.g. 0.25), or a csv file with two columns sample_id, and cut off.
+
+  - <span class="parameter">bool</span><br>
+      You can add a new column to the mudata['rna'].obs with boolean (True/False) values, and then list
+      that column under this bool section. This can be done for any modality.
+
+<span class="parameter">var</span><br>
+    Parameters for var, i.e. gene (feature) level filtering:
+
+  - <span class="parameter">min</span><br>
+
+    - <span class="parameter">n_cells_by_counts</span> `Integer`<br>
+
+  - <span class="parameter">max</span><br>
+
+    - <span class="parameter">total_counts</span> `Integer`<br>
+
+    - <span class="parameter">n_cells_by_counts</span> `Integer`<br>
+
+### Protein-specific filtering (prot)
```

Comment:
Should we still list the entire structure (obs: min / max, var: min / max) here, although there are no specific parameters in the respective sections?